README.md

```diff
@@ -107,6 +107,9 @@ If you want karma to automatically rerun the tests on file modification, use:
 
     ./script/karma start
 
+### Multilingual
+Do not forget to run `rake tmp:cache:clear` after locales are updated to reload I18n js translations.
```

Comment:
Does this need to be included in any deployment scripts, or is restarting the server sufficient?